src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -463,10 +469,13 @@ public FormValidation doCheckProjectKey(@QueryParameter String value) throws IOE
             return FormValidation.ok();
         }
 
-        public ListBoxModel doFillPriorityIdItems() {
+        public ListBoxModel doFillPriorityIdItems(@AncestorInPath final Item item) {
             ListBoxModel items = new ListBoxModel().add(""); // optional field
-            for (JiraSite site : JiraGlobalConfiguration.get().getSites()) {
-                JiraSession session = site.getSession();
+            ItemGroup parent = item == null ? null : item.getParent();
```

Comment:
Those two next lines are unreadable for me, I'd prefer if/else here or at least have the parentheses around the ternary condition.